project2/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Project 2: C O M P S </title>
+    <link href="https://fonts.googleapis.com/css?family=Be+Vietnam:300,600,800&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+    <section>
+        <img src="images/featured-film-example.jpg" class="articlepicture"
```

Comment:
Remember to resize and optimize your images. This file is 1200px wide, but since your `section` container is 400px, it's 3x larger than it needs to be. Additionally, [tinyPNG](https://tinypng.com/) says it can be optimized for a filesize savings of 30%. As you can imagine, on a site with many articles on a single page, those savings would add up, especially for mobile users.